File: amplify/functions/backend/lambda_function.py
from backend_logic import process_request

def handler(event, context):
    """
    AWS Lambda handler
    """
    print("Received event:", event)
    result = process_request(event)
    return {
        "statusCode": 200,
        "headers": {"Content-Type": "application/json"},
        "body": result
    }

File: amplify/functions/backend/lambda-function.js
import { processRequest } from "./backend-logic.js";

export async function handler(event) {
  console.log("Received event:", event);

  const result = await processRequest(event);

  return {
    statusCode: 200,
    headers: { "Content-Type": "application/json" },
    body: JSON.stringify(result),
  };
}
